### ISSUE TITLE
Cannot get /

### ISSUE THREAD

**Issue by Sylvesterx1** [medusajs/medusa#5545]
After fresh installation medusa server is automatic running and i have access to admin panel on port /7001

After close installation and run server again:

npm start -- --port 7001

or with CLI
medusa develop

or
npm install --legacy-peer-deps -- --port 7000
or
npm start --legacy-peer-deps -- --port 7001

info:    Using fake Redis
✔ Models initialized – 27ms
✔ Plugin models initialized – 56ms
✔ Strategies initialized – 23ms
⠋ Initializing modules
✔ Modules initialized – 4ms
✔ Database initialized – 67ms
✔ Repositories initialized – 21ms
✔ Services initialized – 42ms
✔ Express intialized – 3ms
✔ Plugins intialized – 273ms
✔ Subscribers initialized – 18ms
✔ API initialized – 60ms
⠋ Initializing defaults
warn:    You don't have any notification provider plugins installed. You may want to add one to your project.
✔ Defaults initialized – 61ms
⠋ Initializing search engine indexing
✔ Indexing event emitted – 14ms
✔ Server is ready on port: 7001 – 16ms
::1 - - [03/Nov/2023:07:22:56 +0000] "GET / HTTP/1.1" 404 139 "-" "curl/7.61.1"

output CURL:

[root@vim1999 ~]# curl http://localhost:7001
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>Cannot GET /</pre>
</body>
</html>
[root@vim1999 ~]#

The same for all ports. How to resolve?

**Comment by kasperkristensen** [comment]
Hey @Sylvesterx1

When you run `medusa develop` it will per default run your Medusa server on port 9000 and a development server for the admin dashboard on port 7001.

When you run `npm start -- --port 7001` you are starting the production build of Medusa on port 7001. In production the admin dashboard development server is not started, and with the default configuration it will instead run on `<your-backend>/app`, so in your case that would be `localhost:7001/app`.

The reason you are getting a 404 is because there is no root endpoint in Medusa, if you did `curl http://localhost:7001/store/products` it would return your products. 

So to summarize if you simply run the develop command with no options, you will get your server on localhost:9000 and the admin dashboard on localhost:7001. When you run the start command with no options you will still get your server on localhost:9000 and your admin dashboard will be on localhost:9000/app. 

Hope that clears it up 😄

**Comment by Sylvesterx1** [comment]
Thank you. Running backend  with medusa develop and storefront with npm start resolved issue. Before I also tryning run both with npm start and backend not work in this way.
